js/app/dashboard/components/TaskList.js

```diff
@@ -73,6 +73,10 @@ class TaskList extends React.Component {
     this.props.unassignTasks(this.props.username, task)
   }
 
+  componentDidUpdate(){
+    console.log('componentDidUpdate')
```

Comment:
Ok so here the goal is to clear `preExpanded` (i.e reset it to an empty array). 

1 - Add a new Redux action specifically for this (for example, `CLEAR_PRE_EXPANDED`)
2 - Call this Redux action in `componentDidUpdate`
3 - In the reducer function, when this action is triggered, it should clear the `preExpanded` array
```js
return {
  ...state,
  preExpanded: [],
}
```